### PR TITLE
Eventing API docs for v0.13.0

### PR DESCRIPTION
**Pull request by ellenevans**
Updates to Eventing API docs for v0.13.0 
  * eventing-contrib.md
  * eventing.md

-
-
-

### CODE REVIEW

File: docs/reference/eventing/eventing-contrib.md [hunk 1]
```diff
@@ -2118,12 +2091,12 @@ for the PrometheusSource.</p>
...
 </tbody>
 </table>
-<h3 id="sources.eventing.knative.dev/v1alpha1.SecretValueFromSource">SecretValueFromSource
+<h3 id="sources.knative.dev/v1alpha1.SecretValueFromSource">SecretValueFromSource
 </h3>
 <p>
 (<em>Appears on:</em>
-<a href="#sources.eventing.knative.dev/v1alpha1.KafkaSourceSASLSpec">KafkaSourceSASLSpec</a>, 
-<a href="#sources.eventing.knative.dev/v1alpha1.KafkaSourceTLSSpec">KafkaSourceTLSSpec</a>)
+<a href="#sources.knative.dev/v1alpha1.KafkaSourceSASLSpec">KafkaSourceSASLSpec</a>,
```
Comment (mattmoor) [inline docs/reference/eventing/eventing-contrib.md]: Remove trailing whitespace:
```suggestion
<a href="#sources.knative.dev/v1alpha1.KafkaSourceSASLSpec">KafkaSourceSASLSpec</a>,
```

**Comment by ellenevans** [comment]
The link is fine.

**Pull request merged**